core/dbt/artifacts/resources/v1/components.py

```diff
@@ -181,7 +181,6 @@ class ParsedResource(ParsedResourceMandatory):
     docs: Docs = field(default_factory=Docs)
     patch_path: Optional[str] = None
     build_path: Optional[str] = None
-    deferred: bool = False
```

Comment:
Confirming this is a minor change and safe to make here from a runtime serialization/deserialization perspective 👍 

Additionally, we're not using this field for anything functional in dbt-core's implementation, it was previously just accessed to assert certain behaviour had occurred during deferral.

Reply:
Approving minor schema evolution changes